### ISSUE TITLE
[A11y] Collection Landing - Mobile - Featured Image Thumbnails

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1839]
### Operating System

Windows

### Browser

Edge

### Affected page

Collection Show

### Admin user?

Yes

### Issue Description

The thumbnails, as mentioned previously, don't have a caption of any kind.  Combined with their size and the size of the focus border, this makes for multiple problems:

- Mobile users have no easy way to find out where they'll go clicking on an image.  This was already a bit iffy for screen reader users on desktop, but mobile users with low vision will have a very difficult time since they may not be using a screen reader and won't have any way to hover or focus the images.  The images themselves are so small on a phone that they don't convey any useful information.
- The black focus rectangle is especially problematic on mobile on a dark image.  The thumbnails are extra small on a phone, and the focus rectangle is extra large.  A dark image that only turns black on focus fails contrast requirements.  And the loss of the image is definitely a usability concern by itself.
  - (Some users rely on mobile devices that have an attached keyboard, so this is more relevant than it may sound)

Thumbnails in a phone view:

![image](https://user-images.githubusercontent.com/2157974/128777891-4529e7db-35b4-4c60-b886-1683cfccdbe7.png)

Thumbnails with focus on the last item:

![image](https://user-images.githubusercontent.com/2157974/128777929-bb3a9590-c5b6-4142-ad12-88277bd828f9.png)

### Suggested Fix

As mentioned before, those thumbnails really should stay big enough to be visible and stack vertically instead of being lined up horizontally.  They also need some kind of text describing where they go, and for that text to be available at all times both for screen reader users and mobile users.

This ticket could probably be considered a dupe of a desktop ticket, but there are some mobile concerns that the desktop view really doesn't run into except at pretty absurd zoom levels, and even then, focus is pretty visible because of the size of the images.

**Comment by jechols** [comment]
QA depends on approach taken here.  Possibly trivial to test, but may need expertise.

**Comment by azle** [comment]
@straleyb 

The thumbnail size seems to have been fixed, this is a screen snap from my Apple iPhone. However, on the right side of the thumbnail there seems to be some non clickable squares that look like checkboxes. See screen snap below:

![IMG_2615](https://user-images.githubusercontent.com/8353860/170118893-cfe30749-3c46-497a-bcdb-97c1ef6c0157.PNG)

The thumbnail in desktop and mobile view is missing the black gradient that should appear on hover behind the image name:

![IMG_2617](https://user-images.githubusercontent.com/8353860/170119172-27d5ac8f-c77d-4abb-a558-ec907d9d50c1.PNG)

Here is an example of the black gradient:
<img width="145" alt="Screen Shot 2022-05-24 at 12 43 46 PM" src="https://user-images.githubusercontent.com/8353860/170119306-d10a155e-eedc-48e0-b89f-946f38244bc1.png">
